packages/shared/src/components/auth/OnboardingRegistrationForm.tsx

```diff
@@ -29,6 +32,10 @@ const OnboardingRegistrationForm = ({
 }: OnboardingRegistrationFormProps): ReactElement => {
   const { trackEvent } = useContext(AnalyticsContext);
   const [shouldLogin] = useState(false);
+  const isMobile = useViewSize(ViewSize.MobileL);
+  const socialProofOnboardingMobile: boolean = useFeature(
+    feature.socialProofOnboarding,
+  );
```

Comment:
This will enroll everyone regardless of mobile, it could be ok, just wanted to check what the needed logic would be?

Reply:
Thank you for the heads up